### ISSUE TITLE
Deprecate optimize-module

### ISSUE THREAD

**Issue by manniL** [nuxt-community/modules#222]
### What problem does this feature solve?

https://www.npmjs.com/package/@nuxtjs/optimize is obsolete but still normally listed on NPM.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/modules/issues/c138">#c138</a>)</em></sub></div>

**Comment by manniL** [comment]
cc @pi0

**Comment by pi0** [comment]
@manniL Thanks for notifying that. Agree. We need to either deprecate in the flavor of better alternatives or try to rework some stalled modules like optimize.

**Comment by manniL** [comment]
There isn't even code published anymore (for `@nuxtjs/optimize`) so I guess we can safely deprecate that with a note to the new optimization settings.

Which other stalled modules come to your mind?

**Comment by pi0** [comment]
- optimize
- gtm
- firebase
- offline

Maybe more packages which are not listed in README anymore. But at least these ones should be depricated as there is better alternatives for them.

**Comment by manniL** [comment]
Agreed. We could add an appendix to the README.md in this repo where we can give a small "migration guide" for the few ppl who are still using the modules
